web/components/FcDisplayViewDataAtLocation.vue

```diff
@@ -326,10 +283,14 @@ export default {
     ...mapMutations('requestStudy', [
       'setNewStudyRequest',
     ]),
+    ...mapMutations('viewData', [
+      'removeFilter',
+      'setFilters',
+    ]),
     ...mapMutations([
+      'setDialog',
```

Comment:
can we get rid of `setDialog` altogether (here and elsewhere)?  we're moving away from that to more localized instantiation of dialogs, which helps reduce the amount of state we have to store in `vuex`.